Components/Communication/UARTDriver.cpp

```diff
@@ -17,124 +17,116 @@
 
 // Declare the global UART driver objects
 namespace Driver {
-    UARTDriver uart1(USART1);
-    UARTDriver uart2(USART2);
-    UARTDriver uart3(USART3);
-	UARTDriver uart5(UART5);
-}
+UARTDriver uart1(USART1);
+UARTDriver uart2(USART2);
+UARTDriver uart3(USART3);
+UARTDriver uart5(UART5);
+}  // namespace Driver
 
 /**
  * @brief Transmits data via polling
  * @param data The data to transmit
  * @param len The length of the data to transmit
  * @return True if the transmission was successful, false otherwise
  */
-bool UARTDriver::Transmit(uint8_t* data, uint16_t len)
-{
-	// Loop through and transmit each byte via. polling
-	for (uint16_t i = 0; i < len; i++) {
-		LL_USART_TransmitData8(kUart_, data[i]);
+bool UARTDriver::Transmit(uint8_t* data, uint16_t len) {
+    // Loop through and transmit each byte via. polling
+    for (uint16_t i = 0; i < len; i++) {
+        LL_USART_TransmitData8(kUart_, data[i]);
 
-		// Wait until the TX Register Empty Flag is set
-		while (!LL_USART_IsActiveFlag_TXE(kUart_)) {}
-	}
+        // Wait until the TX Register Empty Flag is set
+        while (!LL_USART_IsActiveFlag_TXE(kUart_)) {}
```

Comment:
why is there a space here, looks weird 

while (..) {
        ^ space

Reply:
what's your issue here. The only thing that changed is the tab spacing.